check.cloudbuild.yaml

```diff
@@ -0,0 +1,18 @@
+steps:
+- name: 'google/cloud-sdk:alpine'
+  entrypoint: 'bash'
+  args:
+  - '-eEuo'
+  - 'pipefail'
+  - '-c'
+  - |-
+    apk add --update shellcheck && rm -rf /var/chache/apk/*
+    shopt -s globstar; shellcheck -o all **/*.sh
+    docker build -t ${_DOCKER_IMAGE} .
+    git clone ${_TEST_REPO} "$$HOME/src"
+    docker run --rm -v "$$HOME/src":/src ${_DOCKER_IMAGE} cast --skipUpload --sourceDir /src
+  env:
+  - 'DOCKER_BUILDKIT=1'
+substitutions:
+  _DOCKER_IMAGE: cast-highlight:latests
```

Comment:
`latests`, oddly similar to `latest` which does have special behaviour, but I've never seen `latests`. Not sure if its meant to be `latest` here or coincidentally named similar.

I see it is copy pasted and not added here in this PR, so I bet it does work all OK as is.

Reply:
It is a typo, indeed. However, it's only for tests, so we ain't push it anywhere.

Reply:
Fixed in https://github.com/tidalmigrations/cast-highlight-docker/commit/9caac71160bcea84e04da9b8d903f889dbd10316